Makefile

```diff
@@ -49,3 +49,25 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+DURATION ?= 3s
+COUNT ?= 5
+
+.PHONY: bench
+bench:
+	rm -rf bench
+	mkdir bench
+	go clean -testcache
+	go test -bench=. -benchmem -benchtime=${DURATION} -count=${COUNT} > bench/bench.txt
+	go get golang.org/x/perf/cmd/benchstat && benchstat bench/bench.txt
+
+.PHONY: ab-perf
```

Comment:
It is unclear for me - what does mean 'ab'?
I know about ["A/B testing"](https://en.wikipedia.org/wiki/A/B_testing), but I suppose it is not about this kind of testing.

Reply:
In my understanding ab testing is about to compare two variants of API and estimate the possible user experience or something like that. So, `perf` postfix would mean that we estimate the performance difference between 2 options and then we have some view of user experience from using both of them. I have renamed that target to `perf-diff`, I think that sounds a bit better for understanding what it does.